CHANGELOG_PENDING.md

```diff
@@ -10,6 +10,7 @@ program](https://hackerone.com/tendermint).
 ### BREAKING CHANGES:
 
 - CLI/RPC/Config
+- [rpc] \#3818 Make `max_body_bytes` and `max_header_bytes` configurable
```

Comment:
Can you move this to Improvements, it is not breaking anything.

Reply:
@marbar3778 
oh, sorry. I fixed it.